app/vlinsert/main.go

```diff
@@ -24,6 +26,18 @@ func Stop() {
 // RequestHandler handles insert requests for VictoriaLogs
 func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 	path := r.URL.Path
+
+	// agents, which do not support custom path prefix
```

Comment:
There is no reason to merge it without custom path prefix support. 

I'd suggest to : 
- add `/insert/datadog` prefix to the vlogs api path
- suggest users to use local proxy with rewrite rule as a target for datadog logs url
- open issue for datadog-agent to add custom prefix support for `http` based clients.

Reply:
there was a PR for a half a year, they are ignoring it

Reply:
I see only one option in this case - provide configuration example with local nginx for datadog agent and path rewrite url for it.

 But more realistically, we shouldn't add datadog ingestion protocol for logs unless issue will be fixed at agent side.

Reply:
i'm not sure this expectation is realistic, also there's no logs agent with datadog output, which supports custom path prefix

Reply:
what is a problem to use this path for datadog? it doesn't intersect with any other protocol

Reply:
Prefix simplifies routing for requests and it will be required for cluster version.

 It also possible to add only TCP based ingestion.

Reply:
TCP is only supported by datadog agent and fluentd plugin

Reply:
For me, it's good option to go. It ll not confuse users with `/insert/datadog` prefix, which is really tricky to change. 

When custom prefix will be implemented at datadog-agent side, we'll add http ingestion handler for at vlogs.